journaled.gemspec

```diff
@@ -29,6 +29,8 @@ Gem::Specification.new do |s|
   s.add_development_dependency "rspec-rails"
   s.add_development_dependency "rspec_junit_formatter"
   s.add_development_dependency "rubocop-betterment", "1.3.0"
+  s.add_development_dependency "spring"
+  s.add_development_dependency "spring-commands-rspec"
```

Comment:
I have `spring` setup with my environment so I added it here so we can get those quicker specs locally